ddd-koans/Entity/Identifier/src/Task.kt

```diff
@@ -0,0 +1,17 @@
+interface Entity {
```

Comment:
not sure if it's worth showing an additional layer of abstraction. On one hand, it enforces to have `id` on other also enforce some common approach. What is more, can't we use `val` and try to parametrised `Entity[IdType]` ? Maybe it's a good candidate to show it later as a possible approach how to enforce common/require entity structure?

Reply:
Yeah, this is a problem with the type here, that it would need to be provided by the user as this can vary, unfortunately.
So I would love to add a generic type in the next iteration or completely remove it.

Reply:
Additionally, this is Scala notation 🤣

Reply:
ah, so I would suggest introducing interface later -> tiny steps

Reply:
I am not sure, but please take a look. I tried to fix it.

Reply:
that's fine